include/wx/qt/glcanvas.h

```diff
@@ -24,6 +24,9 @@ class WXDLLIMPEXP_GL wxGLContext : public wxGLContextBase
 
     virtual bool SetCurrent(const wxGLCanvas& win) const override;
 
+private:
+    QGLContext* m_glContext;
```

Comment:
We should really initialize all members in their declarations, i.e.

```suggestion
    QGLContext* m_glContext = nullptr;
```

Reply:
This one also isn't a common pattern in wx, I didn't see many usages other than `include/wx/private/init.h`.
Suggestion applied though.